flink-connectors/flink-connector-aws-kinesis-data-streams/src/test/resources/profile

```diff
@@ -0,0 +1,7 @@
+[default]
```

Comment:
Nit: Shouldn't this change be on the next commit?

Reply:
This is required by the `AwsV2UtilTest` on the first commit.